doc/integrator/create_application.rst

```diff
@@ -355,66 +353,63 @@ Create the project
 
     .. code:: make
 
+        INSTANCE_ID = <instance>
         INSTANCE = <instance>
-        VARS_FILE = vars_$(INSTANCE).yaml
```

Comment:
Finally all I see from this point is wrong ...
I relay don't know you try to do ...